src/screens/MainApp/Home/Dashboard/NewDiagnoseBoard/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { View, Text, TouchableOpacity } from 'react-native'
+import styles from './styles'
+
+const NewDiagBoard = (props) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.title}>
+        <Text style={styles.text}>
+          ¿Tu planta de Cannabis se ve rara?
+        </Text>
+        <Text style={styles.text}>
+          ¿Parece estar enferma?
+        </Text>
+        <Text style={styles.text}>
+          No te preocupes más
+        </Text>
+      </View>
+      <TouchableOpacity
+        onPress={() => { props.handleRedirectFunc() }}
```

Comment:
The standard js plugin I'm using in visual studio code says that function handlers' name should start with 'handle' word. So it should be onPress={handleSomething} according to it. Suggestion?

Reply:
:thinking: This is the first time I see that suggestion from StandardJS. I think the problem here is about the component itself. I'm not sure if it deserves to be a component. Anyway, how about `handleGoToCreateDiagnoseScreen`?

Reply:
As you said maybe the component itself is the problem. I made it that way because I thought that dashboard could be implemented as an union of independents board components with different functionality.

Reply:
I left it as handleGoToCreateDiagnoseScreen